ush/config_defaults.yaml

```diff
@@ -1997,6 +2007,10 @@ task_run_post:
   # used for post-processing. This is only used if CUSTOM_POST_CONFIG_FILE
   # is set to true.
   #
+  # TESTBED_FIELDS_FN
+  # The file which lists grib2 fields to be extracted to bgsfc for testbed
+  # Empty string means no need to generate bgsfc for testbed
```

Comment:
Would you mind clarifying what bgsfc means here in the comment?

Reply:
bgsfc is an old naming convention that will no longer be used going forward.  I'll change this to something like: "The file which lists grib2 fields to be extracted for testbed files.  Empty string means no need to generate testbed files."